prowler/providers/kubernetes/services/core/core_service.py

```diff
@@ -0,0 +1,83 @@
+from typing import List, Optional
+
+from kubernetes import client
+from pydantic import BaseModel
+
+from prowler.lib.logger import logger
+from prowler.providers.kubernetes.lib.service.service import KubernetesService
+
+
+################## Core ##################
+class Core(KubernetesService):
+    def __init__(self, audit_info):
+        super().__init__(audit_info)
+        self.client = client.CoreV1Api(self.api_client)
```

Comment:
I think this can be done at the `KubernetesService` class, what do you think? Same way as AWS, GCP and Azure.

Reply:
Since it has to be done only once, it is not applicable.